src/utils/openvpn.ts

```diff
@@ -377,7 +375,7 @@ export class VpnManager extends EventEmitter {
 		}
 	}
 
-	public connect(): Bluebird<true> {
+	public async connect(): Promise<void> {
 		return Bluebird.try(() =>
```

Comment:
```suggestion
		await Bluebird.try(() =>
```
and ditch the `.return()`